src/util/dimension.rs

```diff
@@ -0,0 +1,5 @@
+#[derive(Clone, PartialEq, Eq)]
+pub struct Dimension {
+    pub width: String,
+    pub height: String
+}
```

Comment:
Used to specify the dimensions of a brand image. Opted for a generic name for potential reuse in future.